### ISSUE TITLE
BUG - Floor Mods not working anymore , also roads/taxiways are not shown anymore

### ISSUE THREAD

**Issue by modair1000** [lvgamedev/simairport-modding#90]
![image](https://user-images.githubusercontent.com/56348209/80968471-4bc85180-8e18-11ea-9ed7-c82221a280bd.png)
That happens at newer saves , and also party at new games

**Comment by modair1000** [comment]
After i have build new (in-Game) floors , there where the mod floors was , then all other things came back:  like you see here :
![image](https://user-images.githubusercontent.com/56348209/80969324-b201a400-8e19-11ea-955e-8942783be02b.png)
So the problem must be at the floor Mods.   All mods like that not work(not only my ones)

**Comment by modair1000** [comment]
[Floor Pack.zip](https://github.com/lvgamedev/simairport-modding/files/4599264/Floor.Pack.zip)

[Cody Airport 2020 V1.2.airport.zip](https://github.com/lvgamedev/simairport-modding/files/4599271/Cody.Airport.2020.V1.2.airport.zip)

**Comment by modair1000** [comment]
Also after i deleted that hidden  desktop.ini  it not works.
I tested it with a new game 
1. I started a new game (i not change a setting at start)
2. I build a Mod Floor (at all the same)
3. All looks good , like normal 
4. Then the workers build it , but it looks like grass
5. Then parts of the walls are gone (at my old save also the taxiways , roads)
6. If i build then a ingame floor on top of the floor mods , all comes back
![Screenshot (1025)](https://user-images.githubusercontent.com/56348209/81416359-e511a380-9149-11ea-8a63-d049109ec4ba.png)
![Screenshot (1026)](https://user-images.githubusercontent.com/56348209/81416364-e6db6700-9149-11ea-99c1-4a3d8348617a.png)
![Screenshot (1027)](https://user-images.githubusercontent.com/56348209/81416371-e8a52a80-9149-11ea-837e-1f2cc4316425.png)
![Screenshot (1028)](https://user-images.githubusercontent.com/56348209/81416375-e9d65780-9149-11ea-9abf-3b9012e48f48.png)
